folding/validators/protein.py

```diff
@@ -86,6 +84,21 @@ def __init__(
         self.pdb_complexity = defaultdict(int)
         self.epsilon = epsilon
 
+    def setup_system_config(self):
+        # TODO: Add logic to parse parameters to change this...
+
+        def create_random_modifications() -> Dict:
+            sampler = lambda min_val, max_val: round(
+                np.random.uniform(min_val, max_val), 2
+            )
+            kwargs = {"temperature": sampler(100, 500), "friction": sampler(0.8, 1.2)}
```

Comment:
Maybe we should lower the max temperature, the more we increase the temperature the higher the chance of a simulation breaking because of instability. Most proteins break down at 40-60 Celsius which is like 340 Kelvin

Reply:
Resolved. Max Kelvin temp set to 400 by brian